### PR TITLE
Change websockets version to v9

### PR DESCRIPTION
**Pull request by heavenshell**
websocket v9 was released.
https://pypi.org/project/websockets/

I update setup.py's `extra_requirements`

Thank you!

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -51,7 +51,7 @@ def get_packages(package):
 
 
 extra_requirements = [
-    "websockets==8.*",
+    "websockets==9.*",
```
Comment (ssproessig-thales) [inline setup.py]:
```suggestion
    "websockets>=8.*",
```

Comment (ssproessig-thales) [reply setup.py]: wouldn't it make sense to keep 8 as well, as it worked before?

Comment (heavenshell) [reply setup.py]: make sense!
Thank you!

**Review by ssproessig-thales — APPROVED** [review]

**Review by ssproessig-thales — APPROVED** [review]

**Comment by euri10** [comment]
I haven't followed quite closely the changes upstream so let's see what the CI says, !

**Review by euri10 — APPROVED** [review]
the CI says tests pass ! thanks @heavenshell

**Comment by heavenshell** [comment]
Thank you so much!!

**Comment by LincolnPuzey** [comment]
@euri10 @ssproessig-thales Note that websockets9 causes a Deprecation warning:
```
=============================== warnings summary ===============================
tests/protocols/test_websocket.py: 25 warnings
  /home/runner/work/uvicorn/uvicorn/uvicorn/protocols/websockets/websockets_impl.py:95: DeprecationWarning: websockets.handshake is deprecated
    websockets.handshake.check_request(headers)
 ```

I guess this will need to be fixed before websockets10

**Comment by euri10** [comment]
Interesting, I would have thought the pytest configuration we have would have make this fail, since we essentially use all strict settings, or we didn't merge it I'm not sure now, wil check later, thanks for the heads up

**Comment by LincolnPuzey** [comment]
@euri10 Looks like #929 is still open

**Comment by euri10** [comment]
> @euri10 Looks like #929 is still open

grrr ! will try to resurrect this thanks !

**Comment by euri10** [comment]
ok so if we got this merged the CI here would have failed as I thought,
https://github.com/encode/uvicorn/pull/929/checks?check_run_id=2691822451

**Pull request merged**